### ISSUE TITLE
Full Settings screen not viewable for high zoom levels or [horizontally] thin screens

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#21286]
## Description 

If you have your screen level zoomed in and go to Settings, the navigation menu on the side may not be visible. 

If you’re using a standard mouse, no scroll bar appears to move about the screen, so the only way to view the rest of the content is to zoom back out. Additionally, users with horizontally thin screens also encounter this issue.

(Screenshot of `Settings` on high zoom level)
![image](https://user-images.githubusercontent.com/3299098/155431910-f81a80c0-d1a0-429a-996c-4b5d49a93228.png)

(Screenshot of `Settings` with thin screen)
![image](https://user-images.githubusercontent.com/3299098/155431964-d2b8a6ee-1d2c-4996-baa5-d07ceba16539.png)

## Steps to Reproduce

   1. Open Browser and increase zoom level of screen
   2. Open `Settings` page

## Actual result:

Not all Settings menu items are viewable/accessible

## Expected result:
User should be able to view/access all `Settings`

## Reproduces how often: 

Easily

## Brave version (brave://version info)

v.135.103+

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Miscellaneous Information:

User reports: 

- https://community.brave.com/t/no-access-to-saved-passwords-in-version-1-35-101-chromium-98-0-4758-87-official-build-64-bit/344894/10
- https://community.brave.com/t/unable-to-navigate-to-other-areas-in-settings-with-a-small-screen-window/346398

**Comment by ego-lay-atman-bay** [comment]
I'm having a similar issue with the brave shields popup. I have my screen zoom to 125%, and when I click "advanced controls" in the brave shields popup, the bottom is under my taskbar, which I can't access anything under it. 

I think it might not be related to this issue, because the settings page is web based, and the brave shields popup isn't (or at least, it goes outside the window).

**Comment by TheAresjej** [comment]
@Brave-Matt The issue has now been resolved. 😃
![1](https://github.com/brave/brave-browser/assets/19390797/43120211-336c-4261-b623-47d922275dba)
![2](https://github.com/brave/brave-browser/assets/19390797/fdb25839-e25e-43bd-a5a4-91a11e8654be)
Version 1.65.133 Chromium: 124.0.6367.208 (Official Build)

**Comment by rebron** [comment]
Closing per last comment. https://github.com/brave/brave-browser/issues/21286#issuecomment-2110783984
